ui/src/containers/configs/NewConfig.js

```diff
@@ -345,598 +440,590 @@ class NewConfig extends Component {
+                            Add
+                          </a>
+                        </div>
+                      </div>
+                    </div>
                   </>
                 )}
-                {![undefined, ""].includes(this.state.errorMessage) && (
-                  <div className="alert alert-danger mt-4">
-                    <p className="h6 fs-bolder">API response:</p>
-                    {this.state.errorMessage}
-                  </div>
+                {this.state.config["kind"] == "DEPLOYMENT" && (
+                  <>
+                    <div className="col-12 mt-4">
+                      <h5 className="fw-semibold">Deployment config</h5>
+                    </div>
+                    <div className="col-12">
+                      <label className="form-label">Pipeline</label>
+                      <Select
+                        defaultValue={pipelineOptions.find(
+                          (kind) => kind.value === config.spec.pipeline
+                        )}
+                        isSearchable
+                        isClearable
+                        options={pipelineOptions}
+                        onChange={(value) => {
+                          this.handleChange(
+                            "pipeline",
+                            value.value,
+                            "deployment.spec"
+                          );
+                        }}
+                      />
+                    </div>
+                    <div className="col-12 mt-2">
+                      <label className="form-label">Replicas</label>
+                      <input
+                        type="number"
+                        className="form-control"
+                        id="replicas"
+                        min="0"
+                        name="replicas"
+                        onChange={(value) => {
+                          this.handleChange(
+                            "replicas",
+                            value.target.value,
+                            "deployment.spec.replicas"
+                          );
+                        }}
+                        placeholder="1"
+                        step="1"
+                        value={
+                          !isNaN(this.state.config.spec["replicas"])
+                            ? this.state.config.spec["replicas"]
+                            : ""
+                        }
+                      />
+                    </div>
+                  </>
                 )}
-                <div className="col-12 mt-4">
-                  <a
-                    href="/configs/new"
-                    className="btn btn-primary text-white mb-4"
-                    onClick={this.handleCreateConfig}
-                  >
-                    Create config
-                  </a>
-                </div>
+              </>
+            )}
+            {![undefined, ""].includes(this.state.errorMessage) && (
```

Comment:
For this component, it looks like errors are not shown when using the editor, only when using the form.